offchain-modules/packages/scripts/src/integration.ts

```diff
@@ -346,6 +350,7 @@ async function main() {
     multisigConfig,
     ckbStartHeight,
     ethStartHeight,
+    assetManagerContractAddress,
```

Comment:
besides `assetManagerContractAddress`, should deploy `safeMultisignContractAddress` and set `safeMultisignContractAddress` to config